build.sbt

```diff
@@ -55,3 +55,13 @@ lazy val portal = project
     addCommandAlias("testAll", "; test ; it:test")
   )
   .dependsOn(datasource)
+
+//lazy val womtool = project
+lazy val womtool = (project in file("./womtool"))
+  .settings(
+    unmanagedJars in Compile += file("./lib/womtool-48.jar"),
+    name := "womTool",
+    commonSettings,
+    libraryDependencies ++= akkaDependencies ++ testDependencies ++ jsonDependencies ++ macwire ++ testContainers
```

Comment:
Do you really need all these dependencies?

Reply:
you are completely right, thanks!